dist/htmlminifier.js

```diff
@@ -285,8 +237,30 @@
       parseEndTag();
     }
 
-    function parseStartTag( tag, tagName, rest, unary ) {
-      var unarySlash = false;
+    function parseStartTag(input) {
+      var start = input.match(startTagOpen);
+      if (start) {
+        var match = {
```

Comment:
@kangax @XhmikosR you two make up your mind and write it in a new issue - I'll pick it up from there :wink: